ocaml/xapi/xapi_globs.ml

```diff
@@ -1321,6 +1323,11 @@ let other_options =
     , (fun () -> string_of_bool !website_https_only)
     , "Allow access to the internal website using HTTPS only (no HTTP)"
     )
+  ; ( "migration-https-only"
```

Comment:
Do we need this parameter? I would have thought `host.https_only` would be enough for this. This might mean some functionality might break, but it's intended for testing development and it's disable by default anyway

Reply:
I wanted to keep them separate. While we keep `host.https_only`, this config option will go away as soon as we have done full testing. Then HTTPS will be the default for migration, like all other xapi operations.